### PR TITLE
Fix for test cases failing because of 00584fa 

### PR DESCRIPTION
**Pull request by ChaituVR**
**Description of PR**
 Because of this commit: [00584fa](https://github.com/covid19india/covid19india-react/commit/00584fa82e87472952e0a3118af9811281d1e1ae), test cases/specs are failing for all PRs in Github actions as props data object is changed now

**Type of PR**

- [x] Bugfix
- [ ] New feature

**Relevant Issues**  
None

**Checklist**

- [x] Compiles and passes lint tests
- [x] Properly formatted
- [x] Tested on desktop
- [x] Tested on phone

**Screenshots**
Previous:
image

Current:
image

### CODE REVIEW

**Comment by jeremyphilemon** [comment]
cc @tannakartikey

File: src/__tests__/components/Level.test.js [hunk 1]
```diff
@@ -2,30 +2,18 @@ import React from 'react';
...
-  },
-  {
-    active: '5',
-    confirmed: '3',
-    deaths: '1',
-    recovered: '2',
-    state: 'Gujarat',
-  },
-];
+const data = {
```
Comment (tannakartikey) [inline src/__tests__/components/Level.test.js]: Can you change the name of the variable to `state`? As we are passing `states[0]` in the code. 
https://github.com/covid19india/covid19india-react/blob/00584fa82e87472952e0a3118af9811281d1e1ae/src/components/home.js#L122
It explains it better while reading the tests.

**Review by tannakartikey — CHANGES_REQUESTED** [review]
Please mention in your PR description or commit message that we are modifying the test because of this commit: https://github.com/covid19india/covid19india-react/commit/00584fa82e87472952e0a3118af9811281d1e1ae

File: src/__tests__/components/Level.test.js (hunk 1 above)
Comment (ChaituVR) [reply src/__tests__/components/Level.test.js]: Changed them :+1:

**Review by tannakartikey — APPROVED** [review]
LGTM

**Pull request merged**